Server/States/RunningState.cpp

```diff
@@ -134,19 +138,46 @@ void RunningState::CreatePlayers() {
         player->GetPhysicsObject()->SetPhysMat(physics->GetPhysMat("Player"));
         player->GetPhysicsObject()->SetLayer(PLAYER_LAYER);
         player->SetTag(Tag::PLAYER);
-
-        //TODO: clean up
-//        player->GetTransform().SetPosition(Vector3(0,0,0));
         player->GetTransform().SetPosition(currentLevelStartPos);
         player->AddComponent((Component*)(new PlayerMovement(player, world.get())));
 
         playerObjects[pair.first] = player;
     }
 }
 
+void RunningState::StartTriggerVolFunc(int id){
```

Comment:
Don't really need the repeated code in these functions, we can extract it out and pass Replicated::xxxxx as a parameter.

Reply:
I will add this in